README.md

```diff
@@ -1,2 +1,21 @@
-# sync_agile_boards
+# sync-agile-boards
 This is for sync testing
+
+## Return information from ZenHub
+
+Create a virtual environment for Python 3 and activate it, install the requirements, and in a terminal run
+
+```bash
+python src/zenhub.py $PATH_TO_ZENHUB_TOKEN $STORYNUMBER $REPONAME | jq '.'
```

Comment:
it looks like in zenhub.py, the repo name should come before the story number?

Reply:
Good catch @emlys. I corrected it.